packages/devtools_app/lib/src/service/service_extension_widgets.dart

```diff
@@ -841,12 +847,12 @@ abstract class _ServiceExtensionWidget extends StatefulWidget {
   final String Function(Object? error) describeError;
 
   @override
-  _ServiceExtensionMixin<_ServiceExtensionWidget> createState();
+  ServiceExtensionMixin<ServiceExtensionWidget> createState();
 }
 
 /// State mixin that manages calling an async service extension and reports
 /// errors.
-mixin _ServiceExtensionMixin<T extends _ServiceExtensionWidget> on State<T> {
+mixin ServiceExtensionMixin<T extends ServiceExtensionWidget> on State<T> {
```

Comment:
same question here - why change from private to public?

Reply:
same answer